### DISCUSSION TITLE
Uruchomienie sketchu na Arduino Uno

### DISCUSSION THREAD

**Discussion by paku2020** [lkankowski/arduino-multi-relay2#35] [Q&A]
Dzień dobry

Czy sketch powinien działać na arduino uno? Nie mogę skompilować i nie wiem czy ogólnie nie ma możliwości czy tylko u mnie jest problem.

Pozdrawiam

**Comment by lkankowski** [comment]
Działa normalnie. Wiele razy korzystałem z uno do testów.
Następnym razem podaj jakieś komunikaty błędów, to łatwiej będzie pomóc.

niedz., 27 lut 2022, 14:32 użytkownik paku2020 ***@***.***>
napisał:

**Reply by paku2020** [reply]
A w jaki sposób kompilujesz? Wybierasz mega2560?

**Comment by lkankowski** [comment]
Aha, chodzi Ci o brak uno w konfiguracji Platformio.ini :)
Możesz sobie dodać, ale to ten sam mikrokontroler co nano, wiec wybierz
nano.

niedz., 27 lut 2022, 16:50 użytkownik paku2020 ***@***.***>
napisał:

**Comment by klapaucius4** [comment]
Ew. można też sobie dodać do pliku platformio.ini poniższy kod:

```
[env:uno]
platform = atmelavr
board = uno
framework = arduino
build_flags = ${common_env_data.build_flags}
build_unflags = ${common_env_data.build_unflags}
lib_deps = ${common_env_data.lib_deps_builtin}
lib_ignore = ${common_env_data.lib_ignore}
monitor_speed = ${common_env_data.monitor_speed}
monitor_flags = ${common_env_data.monitor_flags}
monitor_filters = ${common_env_data.monitor_filters}
test_ignore = ${common_env_data.test_ignore}
```

Ja tak zrobiłem i dzięki temu mam możliwosć kompilacji na uno.
